deploy/charts/datafuse/values.yaml

```diff
@@ -5,7 +5,7 @@
 replicaCount: 1
 
 image:
-  repository: datafuselabs/fuse-query
+  repository: datafuselabs/datafuse-query
```

Comment:
helm only supports standalone mode, and standalone image name is datafuselabs/datafuse

Reply:
@ZhiHanZ 

If need, you can request a change here.

Reply:
done

Reply:
agreed, helm needs to be refactored